config/routes/nelmio_api_doc.yaml

```diff
@@ -1,3 +1,12 @@
-NelmioApiDocBundle:
-    resource: "@NelmioApiDocBundle/Resources/config/routing.yml"
-    prefix:   /api/doc
+# Expose your documentation as JSON swagger compliant
+app.swagger:
+    path: /api/doc.json
+    methods: GET
+    defaults: { _controller: nelmio_api_doc.controller.swagger }
+
+## Requires the Asset component and the Twig bundle
+## $ composer require twig asset
```

Comment:
Should we require this (twig) separately? It's already installed because of other packages.

Reply:
Actually, I see it has been added.

Reply:
This installs a different package called symfony/twig-pack, I think its different from the twig package we use?

Reply:
The symfony pack is just a bundle/package of commonly used bundles/packages that Symfony uses when using twig.